app/src/main/kotlin/cz/covid19cz/erouska/ui/main/MainActivity.kt

```diff
@@ -77,7 +77,7 @@ class MainActivity :
         viewModel.serviceRunning.observe(this, Observer { isRunning ->
             ContextCompat.getColor(
                 this,
-                if (isRunning && passesRequirements()) R.color.green else R.color.red
```

Comment:
@davidvavra pls review this if correct

Reply:
Vlasta psal, že pokud nejsou splněné requirements (Bluetooth a polohové služby), tak kartička o zapnutí eRoušky tam nemá být vůbec @vandus